### ISSUE TITLE
cat command doesn't work (403 on domain)

### ISSUE THREAD

**Issue by angeloanan** [RayzrDev/SharpBot#133]
HTTP 403 on http://aws.random.cat/
HTTP 403 on http://aws.random.cat/meow

**Comment by Rayzr522** [comment]
Huh, I moved over to `aws.random.cat` from `random.cat` per the instructions from `random.cat/meow`:

```json
{
    "error": "Please start calling the API on the aws sub-domain, http:\/\/aws.random.cat\/meow -- this API serves over 5 million requests a month"
}
```

However, apparently the AWS site is down as well. Without any further instructions as to what subdomain / endpoint to use, it seems like //cat will just have to remain broken for now. Presumably the owner of random.cat will patch this soon, but if not, then I will have to find a new cat image API at some point.

**TL;DR** for now I think I'm just gonna leave this, and we'll see if the owner patches it :)

**Comment by angeloanan** [comment]
Use `thecatapi.com` as an alternative? The docs says that it doesn't require an API key.

**Comment by Madzahttr** [comment]
aws.random.cat works. but its being weird, works fine in the browser for me, but if i curl it, it spits a 403 error

**Comment by angeloanan** [comment]
You sure about that? 
![image](https://user-images.githubusercontent.com/2129163/38455953-07579fca-3aa9-11e8-9bf1-c17f1fa72b9c.png)

**Comment by Rayzr522** [comment]
It's working sometimes and for some people. The origin of the requests' country seems to be playing a part in it as well, perhaps, though that may have been blind luck.

The problem with `TheCatAPI.com` is that it doesn't return data we can use quite as easily as `random.cat/meow`, though we should be able to parse the XML format (`http://thecatapi.com/api/images/get?format=xml`). I'm going to make a branch on SharpBot to test this :)

**Comment by Rayzr522** [comment]
Switched over to thecatapi.com as aws.random.cat seems to be simply far too unstable for now. Please update via `//exec -s git pull`.
